pkg/controller/jenkins/jenkins_controller.go

```diff
@@ -478,6 +478,14 @@ func (r *ReconcileJenkins) setDefaults(jenkins *v1alpha2.Jenkins, logger logr.Lo
 		jenkins.Spec.Master.SecurityContext = &securityContext
 	}
 
+	if jenkins.Spec.Master.SeedJobAgentExecutors < 1 {
+		logger.Info("Setting default seed job agents executors amount")
```

Comment:
...("Setting a default seed job agents executors number of: %d", defaultExecutorsAmount)